pkg/plugins/golang/v4/scaffolds/internal/templates/test/e2e/test.go

```diff
@@ -101,6 +101,7 @@ var _ = Describe("controller", Ordered, func() {
 			By("installing CRDs")
 			cmd = exec.Command("make", "install")
 			_, err = utils.Run(cmd)
+			ExpectWithOffset(1, err).NotTo(HaveOccurred())
```

Comment:
Thank you for the contribution
You need to run `make generate` so that will update the samples under testdata

Reply:
@camilamacedo86 thanks, I've updated.